nemo_skills/evaluation/top_answer_processor.py

```diff
@@ -0,0 +1,314 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
I'd just call it "process_answers" or "aggregate_answers". Shorter and not-less descriptive in my opinion

Reply:
I like `aggregate_answers`. Changing to that.